src/utils.jl

```diff
@@ -24,12 +24,11 @@ macro all_dimensions(f, l...)
 end
 
 Base.float(q::Quantity{T}) where {T<:AbstractFloat} = convert(T, q)
-Base.convert(::Type{T}, q::Quantity) where {T<:Real} =
-    let
```

Comment:
FYI I wrote it like this just for readability purposes. Having all `Base.(f)` functions in a single column makes it easier to scan through things, rather than having some of them be `function Base.(f)` which breaks the formatting. What do you think?

I think I would either prefer all of them to be `function Base.(f)`, or all of them to be `Base.(f) = `. But if you feel strongly about this I'm happy to change.

Reply:
My general rule is that if it's more than 1 line, it should be a function, but I don't feel strongly about it.